components/renku_data_services/message_queue/db.py

```diff
@@ -41,7 +41,9 @@ async def send_pending_events(self) -> None:
 
         async with self.session_maker() as session:
             # we only consider events older than 5 seconds so we don't accidentally interfere with an ongoing operation
-            stmt = select(schemas.EventORM).where(schemas.EventORM.timestamp < datetime.utcnow() - timedelta(seconds=5))
+            stmt = select(schemas.EventORM).where(
+                schemas.EventORM.timestamp_utc < datetime.utcnow() - timedelta(seconds=5)
+            )
             result = await session.execute(stmt)
             events_orm = result.scalars().all()
```

Comment:
Side question: why not use `scalars()` instead of `execute()`? It directly provides scalars results.

```python
result = await session.scalars(stmt)
events_orm = result.all()
```